src/Compilers/Core/CodeAnalysisTest/Emit/CustomDebugInfoTests.cs

```diff
@@ -587,6 +587,62 @@ public void DynamicLocals()
         }
 
         [Fact]
+        public void TupleElementNames()
+        {
+            var builder = new BlobBuilder();
+            var cdiEncoder = new CustomDebugInfoEncoder(builder);
+
+            cdiEncoder.AddTupleElementNames(new[]
+            {
+                ("a", 1, 0, 0, ImmutableArray.Create("e")),
```

Comment:
It would be good to name those elements for readability. But then again, the names won't be verified :-(